backend/Controllers/MetadataController.cs

```diff
@@ -40,7 +55,8 @@ public async Task<IEnumerable<MetadataResource>> GetAllAsync()
 
 		/// <summary>
 		/// Returns a single metadata type, and all its associated metadata entries.
-		/// </summary> 
+		/// </summary>
+        [AllowAnonymous]
```

Comment:
What's up with the indentions

Reply:
Seems to be related to spaces vs. tabs. Not possible to reproduce locally.